### PR TITLE
SVG Export

### LINKED ISSUE #96: SVG Export
## Problem
It could be useful to use the drawings in some external documents, so we need to export the current drawing in a vector based format.

## Solution
Add ability to export diagrams as SVG files.

### PR DESCRIPTION
**Pull request by 0x4d**
Closes #96 

Adds SVG serializer and export behavior

### CODE REVIEW

File: test/features/export/Export.spec.js [hunk 1]
```diff
@@ -133,5 +136,20 @@ describe('modules/export - Export', () => {
...
+        it('should fire svg export', () => {
+            eventBus.on('export', (context) => {
+                const parser = new DOMParser();
+                const doc = parser.parseFromString(
+                    context.payload,
+                    context.mimeType
+                );
+
+                expect(doc.documentElement).toBeDefined();
+                expect(doc.documentElement.nodeName).toBe('svg');
```
Comment (tim-kilian) [inline test/features/export/Export.spec.js:149]: I dont get what this test is ensuring.

**Review by tim-kilian — APPROVED** [review]
Good changes for a simple svg export. Since we havn't one yet this is an improvement and I'll merge it.

File: test/features/export/Export.spec.js (hunk 1 above)
Comment (0x4d) [reply test/features/export/Export.spec.js:149]: It ensures that the behavior returns an XML document that can be parsed and has a root node named svg. There is certainly room for improvement, like making sure the document contains all elements. This is not yet checked for any of the export behaviors.

Comment (tim-kilian) [reply test/features/export/Export.spec.js:149]: Yeah, also the expects are in the callbacks. This leads to the behavior that the event doesn't need to run to succeed the test, which is not what I would expect. There is missing something to ensure, that the callback is running and executing the expects inside the callback.

Comment (0x4d) [reply test/features/export/Export.spec.js:149]: You are absolutely right. Will be fixed in #101.

**Pull request merged**